### PR TITLE
Dan card styling

### PR DESCRIPTION
**Pull request by dweverson**
-Listings card styled.

-Matches card added and styled with confetti affect. Chat button opens new chatroom between matched users. 

-All listings no longer shows logged in users items.

### CODE REVIEW

File: src/components/ListingCard.svelte [hunk 1]
```diff
@@ -26,36 +26,60 @@
       updatedListingLikes = [item, ...updatedListingLikes];
     });
     listingsWithLikes = updatedListingLikes;
+    console.log(listingsWithLikes);
```
Comment (CatHoang) [inline src/components/ListingCard.svelte]: Console log left in, not sure if it's needed.

**Review by CatHoang — COMMENTED** [review]
Just a console log but apart from that, it all looks good!

File: src/components/ListingCard.svelte (hunk 1 above)
Comment (dweverson) [reply src/components/ListingCard.svelte]: Removed

**Review by CatHoang — APPROVED** [review]
Great work

**Pull request merged**